synapse/tests/test_cortex.py

```diff
@@ -852,19 +853,29 @@ async def test_cortex_storm_dmon_log(self):
 
         async with self.getTestCore() as core:
 
-            iden = await core.callStorm('''
-                $que = $lib.queue.add(foo)
-
-                $ddef = $lib.dmon.add(${
-                    $lib.print(hi)
-                    $lib.warn(omg)
-                    $que = $lib.queue.get(foo)
-                    $que.put(done)
-                })
+            with self.getStructuredAsyncLoggerStream('synapse.storm.log',
```

Comment:
did you want to doc $auto in the automation section (or somewhere)?

Reply:
I'll add some docs.

Reply:
added